circuit-codegen-annotations/build.gradle.kts

```diff
@@ -86,4 +86,7 @@ kotlin {
   }
 }
 
-android { namespace = "com.slack.circuit.codegen.annotations" }
+android {
+  namespace = "com.slack.circuit.codegen.annotations"
+  defaultConfig { consumerProguardFiles("consumer-proguard-rules.txt") }
```

Comment:
We'll want this in the JVM artifact too, so let's just put them in `resources/META-INF/proguard/` instead for JVM/android

Reply:
Just pushed a fix where I moved the file in `"src/commonJvmMain/resources/META-INF/proguard/circuit-codegen-annotations.pro"`. I don't have much experience with KMM project so let me know if there's a better way of doing it.